src/widget/wmainmenubar.cpp

```diff
@@ -0,0 +1,692 @@
+#include "widget/wmainmenubar.h"
+
+#include <QDesktopServices>
+
+#include "controlobjectslave.h"
+#include "defs_urls.h"
+#include "mixer/playermanager.h"
+#include "util/cmdlineargs.h"
+#include "util/experiment.h"
+#include "vinylcontrol/defs_vinylcontrol.h"
+
+namespace {
+
+QString buildWhatsThis(const QString& title, const QString& text) {
```

Comment:
(The goal is to build a whats this without introducing any other strings that need translating, so we concatenate the title and the description of hte action)